core/Plugin/ViewDataTable.php

```diff
@@ -455,7 +455,12 @@ private function overrideViewPropertiesWithQueryParams()
     protected function getPropertyFromQueryParam($name, $defaultValue)
     {
         $type = is_numeric($defaultValue) ? 'int' : null;
-        return Common::getRequestVar($name, $defaultValue, $type);
+        $value = Common::getRequestVar($name, $defaultValue, $type);
+        // convert comma separated values to arrays if needed
+        if (is_array($defaultValue)) {
+            $value = Piwik::getArrayFromApiParameter($value);
+        }
+        return $value;
```

Comment:
Will this affect other parameters as well? Eg, the 'label' parameter or a parameter like it might have commas in it but might not be an array.

Reply:
it shouldn't, as those parameters should not have an array as default value.

Reply:
Ah, I see, nvm then.